fieldservice_route/models/fsm_route_dayroute.py

```diff
@@ -13,7 +13,12 @@ class FSMRouteDayRoute(models.Model):
     _description = "Field Service Route Dayroute"
 
     name = fields.Char(required=True, copy=False, default=lambda self: _("New"))
-    person_id = fields.Many2one(comodel_name="fsm.person", string="Person")
+    person_id = fields.Many2one(
+        comodel_name="fsm.person",
+        string="Person",
+        compute="_compute_person_id",
+        store=True,
```

Comment:
This should be `readonly=False` for having a computed writable field and editable like before.